ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -90,6 +90,34 @@ import RevenueCat
         Purchases.logLevel = level
     }
 
+    @objc public static func setLogHander(completion: @escaping ([String: String]) -> Void) {
```

Comment:
Maybe document the format of the dictionary?

Reply:
Actually why a dictionary instead of 2 parameters?

Reply:
I want it to be called by all hybrids like this https://github.com/RevenueCat/purchases-flutter/pull/591/files#diff-a84fbe19c95708058ab36b565eb31eb4a3071f224431bdb07085001b0f6271bbR546-R552

Otherwise I would have to do the mapping to a json in every single hybrid, this way it's done in phc and we don't duplicate code. Similar to what we do with other objects, like CustomerInfo

Reply:
Okay that makes sense 👍🏻 

~~Curious, why lowercase though? I've made the associated `String` be uppercase everywhere to match the one in iOS too. Should we use uppercase?~~

Never mind, I see it's already uppercase, but the comment is backwards :P